Sources/MintKit/Mint.swift

```diff
@@ -188,9 +199,90 @@ public struct Mint {
         try addPackage(git: packagePath.gitPath, path: packagePath.packagePath)
         print("🌱  Installed \(package.commandVersion)".green)
 
+        if global {
+            try installGlobal(packagePath: packagePath)
+        }
+
+
         try? packageCheckoutPath.delete()
     }
 
+    func installGlobal(packagePath: PackagePath) throws {
+
+        let installPath = installsPath + packagePath.package.name
+        try installsPath.mkpath()
+
+        // copy to global installs
+        try? installPath.delete()
+        let output = main.run(bash: "ln -s \(packagePath.commandPath.absolute().string) \(installPath.absolute().string)")
+        guard output.succeeded else {
+            print("🌱  Could not install \(packagePath.package.commandVersion) globally")
+            return
+        }
+
+        let exportCommand = "export PATH=\"\(installsPath):$PATH\""
+
+        // add $PATH to current shell
+        // FIXME: this doesn't seem to work
+        main.run(bash: exportCommand)
+
+        // add $PATH to common files
+        let exportSection = "# Register Mint install path\n\(exportCommand)"
+
+        let files = [
+            "~/.bash_profile",
+            "~/.bashrc",
+            "~/.zshenv",
+            ]
+
+        try files.forEach {
+            let file = Path($0).absolute()
+            if file.exists {
+                let existingString: String = try file.read()
+                if !existingString.contains(exportSection) {
+                    let newString = "\(existingString)\n\n\(exportSection)"
+                    try file.write(newString)
```

Comment:
Probably worth logging out that this is happening, but this makes sense to me 👍

Reply:
done